File: spec/cpp_stl/test_zlib_with_header_78.cpp
#include <boost/test/unit_test.hpp>

#include <zlib_with_header_78.h>

#include <iostream>
#include <fstream>
#include <vector>

BOOST_AUTO_TEST_CASE(test_zlib_with_header_78) {
    std::ifstream ifs("src/zlib_with_header_78.bin", std::ifstream::binary);
    kaitai::kstream ks(&ifs);
    zlib_with_header_78_t* r = new zlib_with_header_78_t(&ks);

    BOOST_CHECK_EQUAL(r->data(), "a quick brown fox jumps over");

    delete r;
}

File: spec/python/test_zlib_with_header_78.py
import unittest

from zlib_with_header_78 import ZlibWithHeader78

class TestZlibWithHeader78(unittest.TestCase):
    def test_zlib_with_header_78(self):
        r = ZlibWithHeader78.from_file("src/zlib_with_header_78.bin")

        self.assertEqual(r.data, "a quick brown fox jumps over")
